examples/docusaurus-2/README.md

```diff
@@ -0,0 +1,33 @@
+# Website
```

Comment:
I think this README.md is supposed to have a deploy button and follow this pattern:

https://github.com/zeit/now/blob/master/examples/gatsby/README.md

https://github.com/zeit/now/blob/master/examples/hugo/README.md